dali/operators/reader/tfrecord_reader_op.cc

```diff
@@ -27,16 +27,17 @@ DALI_REGISTER_OPERATOR(_TFRecordReader, TFRecordReader, CPU);
 DALI_SCHEMA(_TFRecordReaderBase)
   .DocStr(R"code(Read sample data from a TensorFlow TFRecord file.)code")
   .AddArg("path",
-      R"code(List of paths to TFRecord files.)code",
+      R"code(List of paths to the TFRecord files.)code",
```

Comment:
I don't think it's even correct to add "the" here....

Reply:
Done